### ISSUE TITLE
OpenApiParameter with enum type not rendered correctly

### ISSUE THREAD

**Issue by hf-kklein** [aliencube/AzureFunctions.Extensions#26]
Using
```c#
[OpenApiParameter("x-my-enum-header" In = ParameterLocation.Header, Type = typeof(MyEnum))]
```
with 
```c#
public enum MyEnum
{
   ONE_OPTION,
   ANOTHER_OPTION
}
```
leads to an OpenApi-Json:
```json
"schema": {
    "type": "integer",
    "format": "int32"
},
```
but should render as
```json
"enum": {
    "ONE_OPTION"
    "ANOTHER_OPTION"
},
```
and thus allow the use of `ONE_OPTION` or `ANOTHER_OPTION` in the Swagger UI instead of just integer values.

**Comment by justinyoo** [comment]
@hf-kklein Thanks for the issue! I'll have a look.

**Comment by justinyoo** [comment]
@hf-kklein Can this `DisplayAttribute` sort out your issue?

https://github.com/aliencube/AzureFunctions.Extensions/blob/dev/src/Aliencube.AzureFunctions.Extensions.OpenApi/Attributes/DisplayAttribute.cs

```
public enum MyEnum
{
    [Display("ONE_OPTION")]
    Option1,

    [Display("ANOTHER_OPTION")]
    Option2,
}
```

Let me know if this helps.

**Comment by hf-kklein** [comment]
@justinyoo 
Sorry, it didn't help. The OpenApi JSON rendered using the DisplayAttribute on enum values does still require an integer:
```json
"schema": {
    "type": "integer",
    "format": "int32"
}
```

**Comment by justinyoo** [comment]
Err, I see. Apparently, it only works for the other things. I'll have a further look.

**Comment by justinyoo** [comment]
@hf-kklein This has been fixed by #33 and will be released soon.
